### PR TITLE
fix(bounties feed): add filters

### PR DESCRIPTION
**Pull request by VERZUOL1**

### CODE REVIEW

File: astro_2.0/components/DaoDetails/DaoDetailsGrid/DaoDetailsGrid.tsx [hunk 1]
```diff
@@ -162,7 +162,9 @@ export const DaoDetailsGrid: FC<DaoDetailsGridProps> = ({
                 </ActionButton>
                 <ActionButton
                   iconName="proposalBounty"
-                  onClick={e => handleClick(e, `/dao/${id}/tasks/bounties`)}
+                  onClick={e =>
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/DaoDetails/DaoDetailsGrid/DaoDetailsGrid.tsx:169]: I would suggest to use routing constants for navigation

File: astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.tsx [hunk 2]
```diff
@@ -1,46 +1,215 @@
...
+          queryName="bountyPhase"
+          list={FEED_OPTIONS}
+          title={t('feed.filters.chooseAFilter')}
+          className={styles.filter}
+        />
+        <div className={styles.container}>
+          {loading ? (
+            <Loader className={styles.loader} />
+          ) : (
+            <>
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.tsx]: `<></>` wrapping looks redundant

File: astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.module.scss [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
+  margin-top: 24px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.bountyInfo {
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.module.scss]: Looks like this style is not being used.

File: astro_2.0/components/DaoDetails/DaoDetailsGrid/DaoDetailsGrid.tsx (hunk 1 above)
Comment (VERZUOL1) [reply astro_2.0/components/DaoDetails/DaoDetailsGrid/DaoDetailsGrid.tsx:169]: fixed

File: astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.module.scss (hunk 3 above)
Comment (VERZUOL1) [reply astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.module.scss]: fixed

File: astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.tsx (hunk 2 above)
Comment (VERZUOL1) [reply astro_2.0/features/Bounties/components/BountiesFeed/BountiesFeed.tsx]: fixed

**Review by dmytro-almighty — APPROVED** [review]

**Pull request merged**